tests/phpunit/class-wc-stripe-webhook-handler-test.php

```
@@ -1378,4 +1378,226 @@ public function test_process_checkout_session_does_not_schedule_metadata_job_on_
+
+		if ( $expected_note_pattern ) {
+			$notes = wc_get_order_notes(
+				[
+					'order_id' => $final_order->get_id(),
+					'limit'    => 1,
+				]
+			);
+			$this->assertNotEmpty( $notes, 'Expected an order note but none was found.' );
+			$this->assertMatchesRegularExpression( $expected_note_pattern, $notes[0]->content );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_process_webhook_mandate_updated is not named in camelCase. (undefined)

(CamelCaseMethodName)